clustermesh-apiserver/main.go

```diff
@@ -35,8 +44,28 @@ func main() {
 		kvstoremeshdbg.RootCmd,
 	)
 
-	if err := cmd.Execute(); err != nil {
+	if err := rootCmd.Execute(); err != nil {
 		fmt.Println(err)
 		os.Exit(1)
 	}
+
+	if cmdRefDir != "" {
+		// Remove the line 'Auto generated by spf13/cobra on ...'
+		rootCmd.DisableAutoGenTag = true
+		if err := doc.GenMarkdownTreeCustom(rootCmd, cmdRefDir, filePrepend, linkHandler); err != nil {
+			fmt.Println(err)
+		}
+		os.Exit(0)
```

Comment:
This doesn't seem to work, because the execution would terminate with an error after `rootCmd.Execute` above when running `clustermesh-apiserver --cmdref <path>`.